config/env.js

```diff
@@ -87,6 +85,7 @@ function getClientEnvironment(publicUrl) {
         WDS_SOCKET_HOST: process.env.WDS_SOCKET_HOST,
         WDS_SOCKET_PATH: process.env.WDS_SOCKET_PATH,
         WDS_SOCKET_PORT: process.env.WDS_SOCKET_PORT,
+        APP_DEFAULT_TAGS: process.env.APP_DEFAULT_TAGS,
```

Comment:
@shannonwells Just a note that you don't have to have this line if you do prefix with `REACT_APP_`. So `REACT_APP_DEFAULT_TAGS` would have just worked thanks to some magic.